review/static/js/src/review.js

```diff
@@ -0,0 +1,62 @@
+/* Javascript for ReviewXBlock. */
+function ReviewXBlock(runtime, element) {
+  $(function ($) {
+    'use strict'
+
+    $('.review-button').on('click', function(event) {
+      var $btn = $(event.currentTarget),
+          $content = $btn.siblings('.review-content'),
+          $iframe = $content.find('iframe'),
+          title,
+          skipLink;
+
+      if (!$btn.hasClass('disable-click')) {
+        // Prevents double clicking which caused issues from seeing the problems
+        $btn.addClass('disable-click');
+
+        setTimeout(function() {
+          $btn.removeClass('disable-click');
+        }, 100);
+
+        // iFrame loads after the button is clicked so there is not
+        // a large amount of loading upon going to the xBlock
+        if ($iframe.attr('src') === '') {
+          $iframe.attr('src', $btn.data('iframe-src'));
+        }
+
+
+        // Prevent issues with cross-origin frames when in studio
+        // (iFrames are hosted on courses.edx.org)
+        if (window.location.hostname.indexOf("studio") == -1) {
```

Comment:
Instead of doing this, I would say it makes more sense to create a separate studio view in the XBlock itself.